faucet-server/Sources/App/Models/Faucet.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Faucet.swift
+//  App
+//
+//  Created by 翟泉 on 2019/5/22.
+//
+
+import Foundation
+import Vapor
+import Fluent
+import FluentMySQL
+
+struct Faucet: Content, MySQLModel {
```

Comment:
I have difficult understand what a faucet model is. Is it to store records of actual capacity sending out to a GitHub user (by using email as their identity)?

Reply:
It a log of the user receiving CKB.